README.md

```diff
@@ -0,0 +1,25 @@
+Terraform Controller
+====================
+
+Introduction
+------------
+
+This is a custom Kubernetes controller designed to run in the kube-system 
```

Comment:
Change this to the `scipian` namespace.

Reply:
This custom controller runs in the kube-system namespace.

Reply:
We can of course change that, but this is the default for the manager.

Reply:
Okay, it's fine for now then. For MVP we should have a work item to move it under the scipian namespace though.